bms/bms_app/static/bms_app/js/bug_view.js

```diff
@@ -136,3 +127,8 @@ $('#project_id').change(function() {
     var selected_proj_id = $(this).val();
     window.location = "/bug_list/?pid="+selected_proj_id;
 });
+
+
+$(document).ready(function() {
```

Comment:
Keep one $(document).ready(function() only. and include all the function which should be added inside .ready function.